### PR TITLE
Fixes #9143: Embedded openssl command doesn't work on aix

### PR DESCRIPTION
**Pull request by peckpeck**
https://www.rudder-project.org/redmine/issues/9143

### CODE REVIEW

**Comment by peckpeck** [comment]
Commit modified

File: rudder-agent/SOURCES/signature.sh [hunk 1]
```diff
@@ -1,7 +1,11 @@
 #!/bin/sh
 
 # To allow Rudder to provide its own version of openssl
-export PATH=/opt/rudder/bin:$PATH
```
Comment (amousset) [inline rudder-agent/SOURCES/signature.sh]: Shouldn't we keep this when not on AIX?

**Comment by peckpeck** [comment]
Commit modified

**Comment by amousset** [comment]
OK, merging this PR

**Pull request merged**